### ISSUE TITLE
order of batch normalize in keras.applications.InceptionV3 ?

### ISSUE THREAD

**Issue by higumachan** [fchollet/keras#4618]
Hi, I'm using "[keras.applications.inception_v3.py](https://github.com/fchollet/keras/blob/master/keras/applications/inception_v3.py)".
I have question in its implementation.
The conv2d_bn implementation is as
```
x = Convolution2D(nb_filter, nb_row, nb_col,
                      subsample=subsample,
                      activation='relu',
                      border_mode=border_mode,
                      name=conv_name)(x)
x = BatchNormalization(axis=bn_axis, name=bn_name)(x)
```
I think BatchNormalization should be before ReLu, and other similar frameworks also have implemented BatchNormalization before ReLu. 
- https://github.com/tensorflow/models/blob/master/inception/inception/slim/ops.py
- https://github.com/smichalowski/google_inception_v3_for_caffe
- https://github.com/dmlc/mxnet/blob/04a077985f6f219bee318dfef8efa182b5bc142e/example/image-classification/symbol/inception-v3.py

I want to ask the reason to keep BN after relu
